src/docs/sphinx/advancedExamples/validationStudies/faultVerification/Example.rst

```diff
@@ -0,0 +1,289 @@
+Solid mechanics solver
+--------------------------
+
+GEOSX is a multi-physics platform. Different combinations of
+physics solvers available in the code can be applied
+in different regions of the domain and be functional at different stages of the simulation.
+The ``Solvers`` tag in the XML file is used to list and parameterize these solvers.
+
+To specify a coupling between two different solvers, we define and characterize each single-physics solver separately.
+Then, we customize a *coupling solver* between these single-physics
+solvers as an additional solver.
+This approach allows for generality and flexibility in constructing multi-physics solvers.
+The order of specifying these solvers is not restricted in GEOSX.
+Note that end-users should give each single-physics solver a meaningful and distinct name, as GEOSX will recognize these single-physics solvers based on their customized names and create user-expected coupling.
+
+As demonstrated in this example, to setup a poromechanical coupling, we need to define three different solvers in the XML file:
+
+- the mechanics solver, a solver of type ``SolidMechanics_LagrangianFEM`` called here ``mechanicsSolver`` (more information here: :ref:`SolidMechanicsLagrangianFEM`),
+
+
+.. literalinclude:: ../../../../../../inputFiles/poromechanics/impermeableFault_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_MECHANICALSOLVER -->
+  :end-before: <!-- SPHINX_MECHANICALSOLVER_END -->
+
+
+- the single-phase flow solver, a solver of type ``SinglePhaseFVM`` called here ``SinglePhaseFlowSolver`` (more information on these solvers at :ref:`SinglePhaseFlow`),
+
+
+.. literalinclude:: ../../../../../../inputFiles/poromechanics/impermeableFault_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SINGLEPHASEFVM -->
+  :end-before: <!-- SPHINX_SINGLEPHASEFVM_END -->
+
+
+- the coupling solver (``SinglePhasePoromechanics``) that will bind the two single-physics solvers above, which is named as ``PoromechanicsSolver`` (more information at :ref:`PoroelasticSolver`).
+
+
+.. literalinclude:: ../../../../../../inputFiles/poromechanics/impermeableFault_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_POROMECHANICSSOLVER -->
+  :end-before: <!-- SPHINX_POROMECHANICSSOLVER_END -->
+
+
+The two single-physics solvers are parameterized as explained
+in their corresponding documents. 
+
+In this example, let us focus on the coupling solver.
+This solver (``PoromechanicsSolver``) uses a set of attributes that specifically describe the coupling process within a poromechanical framework.
+For instance, we must point this solver to the designated fluid solver (here: ``SinglePhaseFlowSolver``) and solid solver (here: ``mechanicsSolver``).
+These solvers are forced to interact with all the constitutive models in the target regions. We specify the discretization method (``FE1``, defined in the ``NumericalMethods`` section), and the target regions (here, we only have one, ``Domain``).
```

Comment:
Can delete this sentence as repetitive.  The discretization stuff is described below.

Reply:
Delete it